src/collect-projects.rst

```diff
@@ -19,6 +19,12 @@ Each Project is identified by a name, icon and color. These are then shown in at
   :alt: Collect's Main Menu with an example Project
   :class: device-screen-vertical
 
+Tapping the Project icon allows you to change or import Project settings, switch between Projects, add a new Project and more:
```

Comment:
This ordering feels odd to me. I'd do "Tapping the Project icon allows you to add, manage, and switch between Projects". I like manage over change or import because it implies deletion which people might want.